pulsar/consumer_partition.go

```diff
@@ -112,6 +112,13 @@ type partitionConsumerOpts struct {
 	maxPendingChunkedMessage    int
 	expireTimeOfIncompleteChunk time.Duration
 	autoAckIncompleteChunk      bool
+	// int failover mode, this callback will be called when consumer change
+	consumerEventListener ConsumerEventListener
+}
+
+type ConsumerEventListener interface {
+	BecameActive(consumer Consumer, partition int32)
```

Comment:
Because a consumer can subscribe to multiple topics, is it possible for a consumer event listener to know which topic this callback invocation corresponds to?

Reply:
Hi @dfarr, thanks for your suggestion. I forgot that we cannot get topic name from `Consumer` in Go client.

Reply:
Try the update, please.

Reply:
Awesome! Thanks so much @labuladong